data/json/items/comestibles/carnivore.json

```diff
@@ -162,5 +187,122 @@
       "price": 1.5,
       "calories": 2.0
     }
+  },
+  {
+    "id": "jerky",
+    "type": "COMESTIBLE",
+    "name": "meat jerky",
+    "name_plural": "meat jerky",
+    "description": "Salty dried meat that never goes bad, but will make you thirsty.",
+    "weight": 40,
+    "volume": 1,
+    "price": 320,
+    "container": "bag_plastic",
+    "material": "flesh",
+    "symbol": "%",
+    "color": "brown",
+    "comestible_type": "FOOD",
+    "charges": 2,
+    "nutrition": 12,
```

Comment:
Could be a good idea to specify it using "copy-from". That way the nutrition could be specified as a percentage of the original nutrition.